x/auth/tx/service.go

```diff
@@ -161,3 +162,14 @@ func RegisterTxService(
 func RegisterGRPCGatewayRoutes(clientConn gogogrpc.ClientConn, mux *runtime.ServeMux) {
 	txtypes.RegisterServiceHandlerClient(context.Background(), mux, txtypes.NewServiceClient(clientConn))
 }
+
+func parseOrderBy(orderBy txtypes.OrderBy) string {
+	switch orderBy {
+	case txtypes.OrderBy_ORDER_BY_ASC:
+		return "asc"
+	case txtypes.OrderBy_ORDER_BY_DESC:
+		return "desc"
+	default:
+		return "asc"
```

Comment:
Not sure whether it's appropriate to comment here as a non-developer. But according to my [previous proposal](https://github.com/tendermint/tendermint/issues/6155) about the default ordering of txs in Tendermint, I was asking to return txs in descending order by default, which is contradicting the statement here, and it would also be impossible to jump to the empty case of the switch statement in QueryTxsByEvents.

Reply:
> Not sure whether it's appropriate to comment here as a non-developer. 

Of course, we appreciate all stakeholders' opinions.

> I was asking to return txs in descending order by default

Ah, you're totally right, I wasn't aware of this change on Tendermint. I suggest the following change: 

```suggestion
		return ""
```

if orderBy is omitted, then default to whatever the Tendermint default is (which is DESC now).